app/src/utils/api.ts

```diff
@@ -188,13 +187,13 @@ export const RepoAPICall = async (
       const data = await response.json();
       return data;
     } else {
-      console.log(`Response from Repo API ${apiUrl} was not a 200`);
-      return undefined;
+      throw new Error(
+        `${response.status}: ${response.statusText || "3xx/4xx error"}`
```

Comment:
Can you prefix the messages with the name of the function? So in this case, "RepoAPICall: ....". I see the errors in the terminal console and the stack trace is helpful but this might make it easier to read both in the console and in cloudwatch if these are being logged there (since the stack trace is not).

Reply:
There are four throws that should have the prefix of their function name.